app.js

```diff
@@ -31,46 +32,38 @@ const config = require("config");
 const session = require("express-session");
 const redis = require("redis");
 
-
-
 const app = express();
 
 app.use(logger("dev"));
-app.use(express.raw({type: 'application/octet-stream', limit: "2gb"}));
-app.use(express.json({limit: "512kb"}));
+app.use(express.raw({ type: "application/octet-stream", limit: "2gb" }));
+app.use(express.json({ limit: "512kb" }));
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use("/demo", express.static("public"));
 app.use("/static/img/logos", express.static("public/img/logos"));
 
 // Security defaults with helmet
 app.use(helmet());
-if(config.get("server.useRedis")){
+if (config.get("server.useRedis")) {
   // Messy but avoids any open file handles.
   const redisClient =
-  process.env.NODE_ENV !== "test"
-    ? redis.createClient({
-        password: config.get(process.env.NODE_ENV + ".redisPass"),
-      })
-    : require("redis-mock").createClient();
+    process.env.NODE_ENV !== "test"
+      ? redis.createClient({
+          url: config.get(process.env.NODE_ENV + ".redisUrl"),
+        })
+      : require("redis-mock").createClient();
   const redisStore = require("connect-redis")(session);
   redisClient.on("error", (err) => {
     console.log("Redis error: ", err);
   });
-  
-  const redisCfg = {
-    host: config.get(process.env.NODE_ENV + ".redisHost"),
-    port: config.get(process.env.NODE_ENV + ".redisPort"),
-    client: redisClient,
-    ttl: config.get(process.env.NODE_ENV + ".redisTTL"),
-  };
+
   app.use(
     session({
       secret: config.get("server.sharedSecret"),
       name: "G5API",
       resave: false,
       saveUninitialized: true,
-      store: new redisStore(redisCfg),
+      store: new redisStore({ client: redisClient }),
```

Comment:
This is being changed in the PR #159 as the project is being updated to be a bit more modern.

Reply:
I think this pr should wait for the #159 to be merged and then re-evaluate the changes in here.